sky-selection-page/index.html

```diff
@@ -5,35 +5,42 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    
     <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="../styles/reset.css">
+    <!-- <link rel="stylesheet" href="../styles/reset.css">
     <link rel="stylesheet" href="../styles/style.css">
-    <link rel="stylesheet" href="../styles/home.css">
+    <link rel="stylesheet" href="../styles/home.css"> -->
+    <link rel="stylesheet" href="./sky.css">
+
+    <link rel="icon" href="../assets/favicon_io/favicon.ico">
+    
     <title>Find your spirit animal</title>
+
 </head>
 
 <body>
     
-    <header id="display-user">
-
+    <header>
+        <div id="current-status"></div>
     </header>
 
     <main>
-        <section class="main-section">     
-            <h2>Visit a constellation</h2>
-            <form class="links">
-                <a class="link" href= "../constellations-page/?id=cassiopeia">Cassieopia</a>
-                <a class="link" href="../constellations-page/?id=ursaMajor">Ursa Major</a>
-                <a class="link" href="../constellations-page/?id=virgo">Virgo</a>
+        <div></div>
+        <section class="main-section">  
+            <form class="links" id="links">
             </form>
+            <h1>Visit the constellations</h1>
         </section>
+        <div></div>
     </main>
+
     <footer>
         <!-- <audio loop autoplay="true" id="space-audio" controls src="https://ia802705.us.archive.org/1/items/LuxAeterna_928/LuxAeternaByClintMansell.ogg"> -->
     </footer>
 
+    <script type="module" src="./create-links.js"></script>
     <script type="module" src="./sky.js"></script>
-   
+    <!-- <script type="module" src="./disable-links.js"></script> -->
```

Comment:
You should have only one javascript file to work with an html file. If you need multiple files they should be made into functions that that you call in your one javascript files. Having multiple files like this is messy code that makes it difficult to read.